src/main/java/com/hubspot/jinjava/lib/filter/DivideFilter.java

```diff
@@ -87,6 +88,9 @@ public Object filter(Object object, JinjavaInterpreter interpreter, String... ar
         throw new InvalidInputException(interpreter, this, InvalidReason.NUMBER_FORMAT, object.toString());
       }
     }
+    if (object instanceof SafeString) {
```

Comment:
Perhaps you could use a Delegate pattern here and implement `SafeStringFilter` which takes care of this rather than manually checking for SafeStrings in every filter?

You could have the default interface implement `filter`, then delegate to a `safeFilter` method in each of these classes, unwrapping and wrapping the `SafeString` around it.

Reply:
Sounds good, thanks!